doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -424,11 +416,34 @@ Response code:
 * Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more details.
 
 + Parameters
-    + limit: 20 (optional, number)
-    + offset: 20 (optional, number)
-    + type: Room (optional, string)
-    + id: Bcn_Welt (optional, string)
-    + idPattern: (optional, string)
+    + id: Boe_Idearium (optional, string) - A comma separated list of elements.
+    Retrieve entities which ID match one of the elements in the list.
+    
+    + type: Room (optional, string) -  comma separated list of elements.
```

Comment:
If we confirm that apiary rendering shows that, then it is ok. Otherwise, I'm afriad we are in a case in which improving one rendering damages the other and that should be avoided.